### DISCUSSION TITLE
Aetna Index File

### DISCUSSION THREAD

**Discussion by moodytheboss** [CMSgov/price-transparency-guide#689] [General]
We are looking into Aetna MRF (2023-01-05_Aetna-Life-insurance-Company_index.json). In the index file, we noticed the same plan (plan_name: Aetna Gold plan_id: 20523CA003) is reported multiple times. Our interpretation of the CMS schema is that one plan needs to be reported once with all relevant in-network files in the in_network_files field. Can someone help me if my interpretation is incorrect or if there are issues with Aetna files? Any input would be highly appreciated. Here is the snapshot of the file:

`{
            "reporting_plans": [
                {
                    "plan_name": "Aetna Gold",
                    "plan_id_type": "HIOS",
                    "plan_id": "20523CA003",
                    "plan_market_type": "group"
                }
            ],
            "in_network_files": [
                {
                    "description": "in-network file",
                    "location": "https://mrf.healthsparq.com/aetnacvs-egress.nophi.kyruushsq.com/prd/mrf/AETNACVS_I/ALICFI/2023-04-05/inNetworkRates/2023-04-05_b4eec29b-39df-4b58-bcd2-7598e4ca2990_Aetna-Health-of-California-Inc..json.gz"
                }
            ],
            "allowed_amount_file": {
                "description": "allowed amount file",
                "location": "https://mrf.healthsparq.com/aetnacvs-egress.nophi.kyruushsq.com/prd/mrf/AETNACVS_I/ALICFI/2023-04-05/allowedAmounts/2023-04-05_b4eec29b-39df-4b58-bcd2-7598e4ca2990_Aetna-Health-of-California-Inc..json.gz"
            }
        },
        {
            "reporting_plans": [
                {
                    "plan_name": "Aetna Gold",
                    "plan_id_type": "HIOS",
                    "plan_id": "20523CA003",
                    "plan_market_type": "group"
                }
            ],
            "in_network_files": [
                {
                    "description": "in-network file",
                    "location": "https://mrf.healthsparq.com/aetnacvs-egress.nophi.kyruushsq.com/prd/mrf/AETNACVS_I/ALICFI/2023-04-05/inNetworkRates/2023-04-05_297ed69c-730a-4a00-bbf6-3626f48fd595_Aetna-Health-of-California-Inc..json.gz"
                }
            ],
            "allowed_amount_file": {
                "description": "allowed amount file",
                "location": "https://mrf.healthsparq.com/aetnacvs-egress.nophi.kyruushsq.com/prd/mrf/AETNACVS_I/ALICFI/2023-04-05/allowedAmounts/2023-04-05_297ed69c-730a-4a00-bbf6-3626f48fd595_Aetna-Health-of-California-Inc..json.gz"
            }
        },
        {
            "reporting_plans": [
                {
                    "plan_name": "Aetna Gold",
                    "plan_id_type": "HIOS",
                    "plan_id": "20523CA003",
                    "plan_market_type": "group"
                }
            ],
            "in_network_files": [
                {
                    "description": "in-network file",
                    "location": "https://mrf.healthsparq.com/aetnacvs-egress.nophi.kyruushsq.com/prd/mrf/AETNACVS_I/ALICFI/2023-04-05/inNetworkRates/2023-04-05_1df7d935-9c22-47c6-bca9-09fcf0c375e9_Aetna-Health-of-California-Inc..json.gz"
                }
            ],
            "allowed_amount_file": {
                "description": "allowed amount file",
                "location": "https://mrf.healthsparq.com/aetnacvs-egress.nophi.kyruushsq.com/prd/mrf/AETNACVS_I/ALICFI/2023-04-05/allowedAmounts/2023-04-05_1df7d935-9c22-47c6-bca9-09fcf0c375e9_Aetna-Health-of-California-Inc..json.gz"
            }
        },`

**Comment by nmayle** [comment]
That doesn't seem to be how the language has been interpreted by others. It is fairly common for payers to "attach" in-network files to a plan via multiple reporting plans objects. The language in the schema does not explicitly say this is valid, but the first example on the page with the schema does show the allowed-amounts file being attached to a plan via a separate reporting plan object than the one that attached the in-network file.

**Reply by moodytheboss** [reply]
Thank you @nmayle for your response. I looked into the example you were referring to, and It does help. Basically, payers can report in-network files associated with the same plan via multiple reporting_plan objects. The above file snippet can also be interpreted as (considering only in_network_files):
`"reporting_plans": [ { "plan_name": "Aetna Gold", "plan_id_type": "HIOS", "plan_id": "20523CA003", "plan_market_type": "group" } ], "in_network_files": [ { "description": "in-network file", "location": "https://mrf.healthsparq.com/aetnacvs-egress.nophi.kyruushsq.com/prd/mrf/AETNACVS_I/ALICFI/2023-04-05/inNetworkRates/2023-04-05_b4eec29b-39df-4b58-bcd2-7598e4ca2990_Aetna-Health-of-California-Inc..json.gz" } , { "description": "in-network file", "location": "https://mrf.healthsparq.com/aetnacvs-egress.nophi.kyruushsq.com/prd/mrf/AETNACVS_I/ALICFI/2023-04-05/inNetworkRates/2023-04-05_297ed69c-730a-4a00-bbf6-3626f48fd595_Aetna-Health-of-California-Inc..json.gz" }, ...]`
